### PR TITLE
Update SQS docs and snippets to core V8

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

File: Snippets/Sqs/Sqs_6/Usage.cs [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+
+class Usage
+{
+    void AssumePermissionsOnPolicy(EndpointConfiguration config)
+    {
+        #region assume-permissions
+
+        var transport = new SqsTransport();
+        
+        transport.Policies.SetupTopicPoliciesWhenSubscribing = false;
```
Comment (timbussmann) [inline Snippets/Sqs/Sqs_6/Usage.cs:22]: does that not work with object initializers?

File: Snippets/Sqs/Sqs_6/Usage.cs [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+        #endregion
+    }
+
+    void MaxTTL(EndpointConfiguration endpointConfiguration)
+    {
+        #region MaxTTL
+
+        var transport = new SqsTransport();
+
+        transport.MaxTimeToLive = TimeSpan.FromDays(10);
```
Comment (timbussmann) [inline Snippets/Sqs/Sqs_6/Usage.cs:126]: in MSMQ I've been using object initializers for these snippets (as that seems to be the code style we should follow now...): https://docs.particular.net/transports/msmq/transportconfig?version=msmqtransport_2

File: transports/sqs/delayed-delivery_enable_sqs_[6,).partial.md [hunk 3]
```diff
@@ -0,0 +1 @@
+In version 6 and higher the unrestricted delayed delivery feature is always enabled but in order to work it has to be enabled in both the sender and the receiver. When communicating with endpoints running older version of the transport make sure to ensure it is enabled.
```
Comment (timbussmann) [inline transports/sqs/delayed-delivery_enable_sqs_[6,).partial.md:1]:
```suggestion
In version 6 and higher the unrestricted delayed delivery feature is always enabled by default. In order to work, it has to be enabled in both the sender and the receiver. When communicating with endpoints running older version of the transport make sure to ensure it is enabled.
```

> When communicating with endpoints running older version of the transport make sure to ensure it is enabled.

That sentence reads a bit weird to me. Why shouldn't it be enabled anyway if I want to make use of it? Why is the fact that communicating an older endpoint relevant when I'm supposed to have it enabled anyway?

**Review by mauroservienti — APPROVED** [review]
Apart from using object initializers where possible, it looks good to me.

File: Snippets/Sqs/Sqs_6/Usage.cs (hunk 1 above)
Comment (mauroservienti) [reply Snippets/Sqs/Sqs_6/Usage.cs:22]: I don't think so 😞

File: transports/upgrades/amazonsqs-5to6.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+## Delayed delivery
+
+The unrestricted delayed delivery is now always enabled so the `UnrestrictedDurationDelayedDelivery()` API has been deprecated.
+
+
+## Configuring the SQS transport
+
+To use the SQS transport for NServiceBus, create a new instance `SqsTransport` and pass it to `EndpointConfiguration.UseTransport`.
```
Comment (mauroservienti) [inline transports/upgrades/amazonsqs-5to6.md:19]:
```suggestion
To use the SQS transport for NServiceBus, create a new `SqsTransport` instance and pass it to `EndpointConfiguration.UseTransport`.
```

File: transports/upgrades/amazonsqs-5to6.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+Use:
+
+```csharp
+var transport = new SqsTransport();
+endpointConfiguration.UseTransport(transport);
+```
+
+## SDK clients
+
+In order to pass customized instances of the SQS and SNS SDK clients to the transport use the `SqsTransport` constructor overload.
```
Comment (mauroservienti) [inline transports/upgrades/amazonsqs-5to6.md:36]:
```suggestion
In order to pass customized instances of the SQS and SNS SDK clients to the transport use the corresponding `SqsTransport` constructor overload.
```

File: transports/upgrades/amazonsqs-5to6.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+snippet: 5to6-clients
+
+## S3 configuration
+
+The S3 usage for large messages is configured via `S3` property of the transport. By default the value is `null` which means S3 usage for sending large messages is disabled.
```
Comment (mauroservienti) [inline transports/upgrades/amazonsqs-5to6.md:42]:
```suggestion
The S3 usage for large messages is configured via the `S3` property of the transport. By default, the value is `null` which means S3 usage for sending large messages is disabled.
```

File: transports/upgrades/amazonsqs-5to6.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+snippet: 5to6-S3
+
+### Encryption
+
+The encryption is configured via `Encryption` property of the S3 settings object. By default the value is `null` which means the messages are not encrypted.
```
Comment (mauroservienti) [inline transports/upgrades/amazonsqs-5to6.md:48]:
```suggestion
Message payload encryption is configured via the `Encryption` property of the S3 settings object. By default, the value is `null` which means the messages are not encrypted.
```

**Review by mauroservienti — CHANGES_REQUESTED** [review]

File: Snippets/Sqs/Sqs_6/Usage.cs (hunk 1 above)
Comment (SzymonPobiega) [reply Snippets/Sqs/Sqs_6/Usage.cs:22]: Nope :(

**Review by mauroservienti — APPROVED** [review]

**Pull request merged**